### DISCUSSION TITLE
Using Crazyflie firmware on own drone (Pixhawk 6C)

### DISCUSSION THREAD

**Discussion by Bartvt21** [bitcraze/discussions#1433] [General]
Hello all,

I am seeking information on using Crazyflie firmware with a custom drone equipped with a Pixhawk 6C flight controller. Specifically, I am interested in understanding the possibilities and limitations of this setup. My goal is to conduct outdoor test flights using Ultra-Wideband (UWB) technology.

As discussed in this [topic](https://github.com/orgs/bitcraze/discussions/1415), the Crazyflie drone is primarily designed for indoor use and may not be robust enough for outdoor environments.

Thanks,

Bart van Tongeren

**Comment by tobbeanton** [comment]
I think porting the Crazyflie firmware to a Pixhawk 6C would be a lot of work, but possible. Easier then would be to build something using [Bolt](https://www.bitcraze.io/2020/10/testing-crazyflie-bolt-and-1-cell-li-ion/) or perhaps wait for the [CF2.1 brushless](https://www.bitcraze.io/2023/10/say-hello-to-the-crazyflie-2-1-brushless/). Hopefully release in Q4.
